pkg/envtest/crd.go

```diff
@@ -190,8 +192,17 @@ func CreateCRDs(config *rest.Config, crds []*apiextensionsv1beta1.CustomResource
 	// Create each CRD
 	for _, crd := range crds {
 		log.V(1).Info("installing CRD", "crd", crd.Name)
-		if _, err := cs.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd); err != nil {
-			return err
+		var err error
+		if _, err = cs.ApiextensionsV1beta1().CustomResourceDefinitions().Get(crd.Name, metav1.GetOptions{}); err != nil {
+			if apierrors.IsNotFound(err) {
```

Comment:
Could we have a test for the "IsAlreadyExists" case succeeding? Other than that lgtm, thanks!
/ok-to-test

Reply:
On it. Thanks for taking the time to review.

Reply:
I ended up adding a re-install step and validation to the existing _install CRDs_ test. Does that suffice?